drivers/sensor/bma280/bma280.c

```diff
@@ -164,6 +164,7 @@ static struct bma280_data bma280_inst_data;
 
 static struct bma280_config bma280_inst_config = {
 	.i2c = I2C_DT_SPEC_INST_GET(0),
+	.int1_gpio = GPIO_DT_SPEC_INST_GET(0, int1_gpios),
```

Comment:
add an #ifdef around `.int1_gpio`

Reply:
note: if we're planning to make drivers multi-instance later (we should), this pattern is better since we can't have #ifdefs inside macros:

```c
static struct bma280_config bma280_inst_config = {
	.i2c = I2C_DT_SPEC_INST_GET(0),
	IF_ENABLED(CONFIG_BMA280_TRIGGER,
		   (.int1_gpio = GPIO_DT_SPEC_INST_GET(0, int1_gpios),))
};
```

Reply:
We certainly should, thanks!